v1/media.yaml

```diff
@@ -0,0 +1,119 @@
+swagger: '2.0'
+info:
+  version: '1.0.0-beta'
+  title: MediaWiki page media API
+  description: API for retrieving data about media items appearing on a given page
+  termsOfService: https://www.mediawiki.org/wiki/REST_API#Terms_and_conditions
+  contact:
+    name: Reading Infrastructure
+    url: https://www.mediawiki.org/wiki/Wikimedia_Reading_Infrastructure_team
+  license:
+    name: Apache licence, v2
+    url: https://www.apache.org/licenses/LICENSE-2.0
+paths:
+  /media/{title}{/revision}:
+    get:
+      tags:
+        - Mobile
```

Comment:
Is it indeed mobile? This seems useful elsewhere.. Maybe put it under pagecontent?

Reply:
Ok, will move to pagecontent.